invenio/modules/formatter/engine.py

```diff
@@ -2018,7 +2018,7 @@ def __init__(self, recID, ln=CFG_SITE_LANG, search_pattern=None,
             # If record is given as parameter
             self.xml_record = xml_record
             self.record = create_record(xml_record)[0]
-            recID = record_get_field_value(self.record, "001") or None
+            recID = int(record_get_field_value(self.record, "001")) or None
```

Comment:
Because of https://github.com/jmartinm/invenio/blob/bibedit-2.0/invenio/modules/formatter/engine.py#L2024

```python
record_get_field_value
```

returns ```str``` so the assertion was failing.